src/elements/element-types.js

```diff
@@ -37,5 +37,9 @@ export default {
 
   RecordLinkElement: 'RecordLinkField',
 
-  StatusElement: 'StatusField'
+  StatusElement: 'StatusField',
+
+  CheckBoxElement: 'CheckBoxField',
+
+  DynamicElementsElement: 'DynamicElementsField'
```

Comment:
Should we just called it `DynamicField`? Simpler and doesn't require us naming something `DynamicElementsElement`?

Would be: `DynamicField` for JSON type, `DynamicElement` for schema model, `DynamicValue` for value model.

Reply:
That sounds good to me.